packages/mui-material/src/Drawer/Drawer.d.ts

```diff
@@ -1,13 +1,104 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
 import { InternalStandardProps as StandardProps, Theme } from '..';
+import { CreateSlotsAndSlotProps, SlotProps } from '../utils/types';
 import { ModalProps } from '../Modal';
+import { BackdropProps } from '../Backdrop';
 import { SlideProps } from '../Slide';
 import { PaperProps } from '../Paper';
 import { TransitionProps } from '../transitions/transition';
 import { DrawerClasses } from './drawerClasses';
 
-export interface DrawerProps extends StandardProps<ModalProps, 'open' | 'children'> {
+export interface DrawerRootSlotPropsOverrides {}
+
+export interface DrawerDockedSlotPropsOverrides {}
+
+export interface DrawerPaperSlotPropsOverrides {}
+
+export interface DrawerTransitionSlotPropsOverrides {}
+
+export interface DrawerBackdropSlotPropsOverrides {}
+
+export interface DrawerSlots {
+  /**
+   * The component used for the Modal root.
+   * @default div
```

Comment:
```suggestion
   * @default Modal
```
i think this should defaulted to `Modal`, as DrawerRoot is built on top of `Modal`

Reply:
Good point, updated.